### PR TITLE
Add missing dependencies.

### PR DESCRIPTION
**Pull request by david-resnick**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -4,9 +4,11 @@
   :license {:name "BSD"
             :url "?"}
   :min-lein-version "2.0.0"
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :dependencies [[org.clojure/clojure "1.8.0"]
```
Comment (bergundy) [inline project.clj:7]: I like that you upgraded clojure

File: project.clj [hunk 2]
```diff
@@ -4,9 +4,11 @@
...
                  [pjstadig/humane-test-output "0.7.1"]
-                 [test2junit "1.2.1"]]
+                 [test2junit "1.2.1"]
+                 [robert/hooke "1.3.0"]
```
Comment (bergundy) [inline project.clj:10]: hooke it not necessary AFAIK

File: project.clj [hunk 3]
```diff
@@ -4,9 +4,11 @@
...
+                 [riemann "0.2.11"]]
```
Comment (bergundy) [inline project.clj:11]: If we add riemann as a dependency here should remove it from the install.sh script

**Comment by david-resnick** [comment]
I missed `install.sh`, I'll close this in favor of a new PR.

**Pull request closed without merging**